man/systemd.journal-fields.xml

```diff
@@ -333,6 +333,16 @@
           </variablelist>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>_SOURCE=</varname></term>
```

Comment:
I think "_SOURCE" is overly generic a term. Maybe call this _STDIO_STREAM= instead? In that case it's very clear that this has a relationship to stdio...